src/modules/tasks/tasks.service.ts

```
@@ -1,107 +1,111 @@
 import { Tasks as Task } from './tasks.model';
 import { groupsService } from '../groups/groups.service';
 import { CustomUser } from '../../common/types/types';
-import { NotFound, Forbidden } from '../../common/exeptions/';
+import {NotFound, Forbidden} from '../../common/exeptions/';
 import { sequelize } from '../../database';
+import {Solution} from '../solutions/solutions.model';
+import {solutionsService} from '../solutions/solutions.service';
+import {Transaction} from 'sequelize';
 
 interface ITasks {
-  groupId: number;
-  taskName: string;
-  fileURL: string;
+    groupId: number;
+    taskName: string;
+    fileURL: string;
 }
 
 const NO_RIGHTS = 'You do not have rights to do this.';
 
 class TasksService {
-  public async findAll(user: CustomUser): Promise<Task[]> {
-    if (!user.isMentor) { 
-      const tasks: Task[] = await Task.findAll({ where: {groupId: user.groupId} });
-      if (!tasks.length) {
-        throw new NotFound(`There is no tasks for group with groupId ${user.groupId}`);
-      } 
-      return tasks;
+    public async findAll(user: CustomUser): Promise<Task[]> {
+        if (!user.isMentor) {
+            const tasks: Task[] = await Task.findAll({ where: {groupId: user.groupId} });
+            if (!tasks.length) {
+                throw new NotFound(`There is no tasks for group with groupId ${user.groupId}`);
```

I don't think that we should throw an exception, please, just return an empty array

---

it's not my code, but I done it.